src/DetailsView/reports/components/automated-check-issue-details-instances.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { NamedSFC } from '../../../common/react/named-sfc';
+import { DecoratedAxeNodeResult } from '../../../injected/scanner-utils';
+
+export interface AutomatedChecksIssueDetailsInstancesProps {
+    nodeResults: DecoratedAxeNodeResult[];
+}
+
+export const AutomatedChecksIssueDetailsInstances = NamedSFC<AutomatedChecksIssueDetailsInstancesProps>(
+    'AutomatedChecksIssueDetailsInstances',
+    ({ nodeResults }) => {
+        const rows: JSX.Element[] = [];
+        nodeResults.forEach((nodeResult, index) => {
+            rows.push(
+                <tr className="row" key={`path-row-${index}`}>
```

Comment:
Looks like this `<tr>` element creation can be extracted to a function... pretty much the same code expect for a name and what you read from `nodeResult`

Reply:
And I'm wondering if each Instance can be extracted to a component. It's a very minor improvement.

Reply:
That can be done, but I am just curios to know if I should make them in a separate file or this file is okay for that?

Reply:
Personally I would prefer that in a separate file. But it's up to you.

Reply:
I'm fine either way.

Reply:
I changed it a bit, let me know if that looks fine to you guys.